php/hyperf/swoole.ini

```diff
@@ -0,0 +1,3 @@
+[swoole]
```

Comment:
no need
this file is already created by `docker-php-ext-enable swoole` in *Dockerfile**

Reply:
@waghanza Hyperf have to set `swoole.use_shortname` to `'Off'`

Reply:
@waghanza Hyperf have to set swoole.use_shortname to 'Off'